### DISCUSSION TITLE
Folder problems ( what to do ?)

### DISCUSSION THREAD

**Discussion by 8-8Sophie8-8** [keiwando/toonsquid#707] [Q&A] [answered]
So I create a folder for a project then realized that I did not place that folder in the practice folder I created. Then I told myself that I simply had to move that folder into the right one.

so I click and select that folder then I saw the ( export option) export obviously mean to move or to move out . 

 Then I click on export nothing happened , click several times nothing, I’m somehow confused and not sure why I cannot move that folder into the right one, am not sure what is going on and why I cannot move that folder.

**Comment by Cat-Butler** [comment]
Possible language mismatch: Export in a software context means "move **_out of the app completely_**" (similar to export for physical items means to move them out of the country) - the same as **_save as_** a separate (external) file (.tsproj project file). Not applicable to folders.

The situation is easier than you imagine: simply tap (do not release) on the project folder, hesitate a second (the folder might change colour or move slightly to indicate it is now "picked up" or active), then drag it into (onto) the practice folder. You might possibly find it easier using a finger rather than the Apple Pencil.

**Comment by 8-8Sophie8-8** [comment]
https://github.com/user-attachments/assets/48611e42-fef6-47f9-be4c-9d4b2fdd137b

I want to get the folder out (the exercise and test folder) to the project folder , can’t get it out of this folder.

**Reply by Cat-Butler** [reply, marked as answer]
Sorry, I understood your original question to be about moving the project folder **_into_** the practice folder. I did not realise you had moved it into a different incorrect folder and needed to move it out.
> So I create a folder for a project then realized that I did not place that folder in the practice folder I created. Then I told myself that I simply had to move that folder into the right one.

To move it out of the current folder, drag the "working on" onto the name of the current folder ("Exercises and test"). You might have to wait a second or two for it to activate (make sure you are precisely positioned on the name). It will then jump you to the next folder level up (i.e. out of the current folder) and you can "drop" the folder or drag it over a different folder before dropping.

**Comment by 8-8Sophie8-8** [comment]
Thanks for the help, I had a hard time with this, it wasn’t working for me and this what I did ( I click and hold the folder  with my right hand then use my left hand to click the arrow to open the previous window then add the folder in the one it was suppose to be) .

it’s because of you that I finally understood how to move out a folder without clicking on the export option which didn’t work, thanks a bunch . :)

https://github.com/user-attachments/assets/d9348081-70ab-4138-bfe4-1d66aac884c7
